src/AppBundle/Entity/Report/Report.php

```diff
@@ -233,12 +233,23 @@ class Report
      */
     private $metadata;
 
+
+    /**
+     * @var Document[]
+     *
+     * @JMS\Type("DateTime")
```

Comment:
no idea why that ended up there :D. removed. Learning now that a second annotation override the first